src/Directory.Build.props

```diff
@@ -5,10 +5,10 @@
     <PackageLicenseUrl>https://github.com/SergiiKram/workflow-core/blob/idempotent-workflow-start/LICENSE.md</PackageLicenseUrl>
     <RepositoryType>git</RepositoryType>
     <RepositoryUrl>https://github.com/SergiiKram/workflow-core</RepositoryUrl>
-    <Version>3.9.0.4</Version>
-    <AssemblyVersion>3.9.0.4</AssemblyVersion>
-    <FileVersion>3.9.0.4</FileVersion>
+    <Version>4.0.0-alpha.1</Version>
+    <AssemblyVersion>4.0.0.0</AssemblyVersion>
+    <FileVersion>4.0.0.0</FileVersion>
```

Comment:
Please don't update versions here, I do it before publishing and to match the original package versioning.

Reply:
reverted